static/css/NISTStyle.css

```diff
@@ -1634,7 +1637,9 @@ pre code{
 
 table{
 	max-width:100%;
-	background-color:transparent
+	background-color:transparent;
+	display: block;

```

Comment:
Changing the display of a fundamental element like `table` seems like a hack to to me, what's the driving reason behind this?

Reply:
Table is ultimately a legacy element that isn't designed for responsiveness. This allows the table's width to be confined to the width of the available horizontal space, and allows the table itself to be scrolled instead of scrolling the whole page, which looks much better from a mobile device.

This is the easiest way I personally know how to accomplish this purely in CSS, and I haven't looked at the markup in this repo to know how easy it would be to go through wrapping these tables with div elements or assigning classes to them.